tests/wakunode2/test_app.nim

```diff
@@ -15,12 +15,13 @@ import
   ../testlib/common,
   ../testlib/wakucore
 
-proc defaultTestWakuNodeConf(): WakuNodeConf =
+proc defaultTestWakuNodeConf*(): WakuNodeConf =
   WakuNodeConf(
     listenAddress: ValidIpAddress.init("127.0.0.1"),
     rpcAddress: ValidIpAddress.init("127.0.0.1"),
     restAddress: ValidIpAddress.init("127.0.0.1"),
     metricsServerAddress: ValidIpAddress.init("127.0.0.1"),
+    dnsAddrsNameServers: @[ValidIpAddress.init("1.1.1.1"), ValidIpAddress.init("1.0.0.1")],
```

Comment:
Both use cloudflare maybe add google in there?

Reply:
Yes! I added only those because they are the ones we configure by default, so tests have a more accurate representation of our nodes.
https://github.com/waku-org/nwaku/blob/6cea737109223bbd2900519d8d958593086b9798/apps/wakunode2/external_config.nim#L115-L118

What do you think is better to do?
Add Google DNS servers to the test anyways, discuss whether to add Google to the default config and if so adding it to the tests, or just leave it the way it is?

Reply:
You are right it should be consistent. Leave it like this it's fine.

Reply:
At the same time, there is this issue: https://github.com/waku-org/nwaku/issues/1490

But yeah, let's keep it until someone actually works on ^